script/docker-compose.yml

```diff
@@ -2,7 +2,7 @@ version: '2.1'
 
 services:
   elasticsearch:
-    image: elasticsearch:alpine
+    image: elasticsearch:5.6-alpine
```

Comment:
Should we use the 6.4 image as we use this in production?

Reply:
sure can do I went to the lowest version that supports the `alias` functionality. It looks to be the same API between v5.x and v6.x but it won't hurt to update it to be sure.